src/meepgeom.cpp

```diff
@@ -480,6 +480,7 @@ double material_grid_val(vector3 p, material_data *md) {
 
 static double tanh_projection(double u, double beta, double eta) {
   if (beta == 0) return u;
+  if ((beta == std::numeric_limits<double>::infinity()) && (u == eta)) return 1.0; // 0*∞ is undefined, so let's just assign 1 here
```

Comment:
This seems sufficient:
```suggestion
  if (u == eta) return 1.0; // 0*∞ is NaN, so let's just assign 1 here
```